config/config.go

```diff
@@ -90,10 +90,13 @@ type MessageBusConf struct {
 type KeyCertConf struct {
 	RootCACertificatePath string `json:"RootCACertificatePath"` // RootCACertificate will be added to truststore
 	PrivateKeyPath        string `json:"PrivateKeyPath"`        // plugin private key
+	PublicKeyPath         string `json:"PublicKeyPath"`         // plugin private key
```

Comment:
rename it to the public key in the comment and whether this variable is used?

Reply:
removed as it is not used for now